### DISCUSSION TITLE
Plating stress

### DISCUSSION THREAD

**Discussion by ghost** [idaholab/moose#22962] [Q&A Modules: Phase field]
Dear all,

I am trying to do electrodeposition on metal substrate.

![geometry](https://user-images.githubusercontent.com/101824131/207659639-26b6be93-19d8-4154-932c-4e93cafaa002.png)

I followed several examples in https://github.com/idaholab/moose/blob/next/modules/combined/examples/phase_field-mechanics. However, it looks like all of them predict stress caused by substrate and electrolyte. At t=0, there is no deposition (or dendrite). How could I calculate stress induced by the deposition on the substrate?

Kinds,

Jung

**Comment by GiudGiud** [comment]
@recuero @jiangwen84
